app/controllers/posts_controller.rb

```diff
@@ -5,12 +5,10 @@ class PostsController < ApplicationController
   before_action :authenticate_user!, except: %i[show index]
 
   def index
-    @posts = Post.all.order('created_at DESC').all
-    @comments = Comment.all.order('created_at ASC').all
+    @posts = Post.all
```

Comment:
Why do you need **all** the posts?

Reply:
thanks
according to point 1. Move the friendship confirmation into the Friendship model, if I move it, it does not work